agogosml_cli/cli/templates/pipeline/azure-ci-app-pipeline.jsonnet

```diff
@@ -30,16 +30,34 @@ local repository = import 'pipeline-repository.libsonnet';
                 "+refs/heads/*"
             ],
             "forks": {
-                "enabled": false,
+                "enabled": true,
                 "allowSecrets": false
             },
             "pathFilters": [
-                "+/sample_app"
+                "+/agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/{{cookiecutter.PROJECT_NAME_SLUG}}"
```

Comment:
This won't get templatized correctly. I'll pushup a fix to your branch.

Reply:
👍 ok, thanks